test/hallread/hallread.ino

```diff
@@ -0,0 +1,11 @@
+void setup() {
+  // put your setup code here, to run once:
+  pinMode(40, INPUT);
+  Serial.begin(57600);
+}
+
+void loop() {
+  // put your main code here, to run repeatedly:
+  
+  Serial.println(digitalRead(40));
+}
```

Comment:
I recommend creating a menu for that which displays 1 or 0 for the sensor like Prusa does for e.g. filament sensor. This way a user does not have to reflash some sketches in order to test the sensor. Also:
- we should not use .ino files because our default IDE is platformio / vsc
- `test` folder is a specially treated folder for [pio unit testing](https://docs.platformio.org/en/latest/plus/unit-testing.html), we should not use it for some utility scripts/sketches.